packages/api-client/src/api/endpoints/customers/createCustomer/index.ts

```
@@ -0,0 +1,18 @@
+import {
+  BigcommerceIntegrationContext,
+  CreateCustomerParameters,
+  CustomerData,
+  CreateCustomerResponse
+} from '../../../../types';
+
+export async function createCustomer(
+  context: BigcommerceIntegrationContext,
+  params: CreateCustomerParameters
+): Promise<CustomerData> {
+  const { client } = context;
+  const { data } = await client.post<
+    CreateCustomerResponse,
+    Array<CreateCustomerParameters>
+  >('/customers', [params]);
+  return data[0];
```

I think here we should provide some props validation. If required props are undefined we should throw an error